recipes/cloud-trace/README.md

```diff
@@ -78,3 +78,10 @@ kubectl apply -f collector-config.yaml
 (This will overwrite any existing collector config, or create a new one if none exists.)
 
 Once the Collector restarts, you should see traces from your application
+
+## View your Spans
```

Comment:
want to add a screenshot?

Reply:
![Screen Shot 2022-10-07 at 4 37 05 PM](https://user-images.githubusercontent.com/3262098/194649254-e75c5313-07e4-44dc-a807-e136a52d30c5.png)

Reply:
done (added it in the example as well)